notebooks/en/_toctree.yml

```diff
@@ -1,69 +1,77 @@
-    title: Annotate text data using Active Learning with Cleanlab
-  - local: rag_with_hugging_face_gemma_mongodb
-    title: Building A RAG System with Gemma, MongoDB and Open Source Models
-  - local: rag_zephyr_langchain
-    title: Simple RAG using Hugging Face Zephyr and LangChain
-  - local: rag_llamaindex_librarian
-    title: RAG "Librarian" Using LlamaIndex
-  - local: pipeline_notus_instructions_preferences_legal
-    title: Create a legal preference dataset
-  - local: semantic_cache_chroma_vector_database
-    title: Implementing semantic cache to improve a RAG system.
-  - local: structured_generation
-    title: RAG with source highlighting using Structured generation
+    - title: LLM and RAG Recipes with other Libraries
+      isExpanded: false
+      sections:
+        - local: issues_in_text_dataset
+          title: Detecting Issues in a Text Dataset with Cleanlab
+        - local: annotate_text_data_transformers_via_active_learning
+          title: Annotate text data using Active Learning with Cleanlab
+        - local: rag_with_hugging_face_gemma_mongodb
+          title: Building A RAG System with Gemma, MongoDB and Open Source Models
+        - local: rag_zephyr_langchain
+          title: Simple RAG using Hugging Face Zephyr and LangChain
+        - local: rag_llamaindex_librarian
+          title: RAG "Librarian" Using LlamaIndex
+        - local: pipeline_notus_instructions_preferences_legal
+          title: Create a legal preference dataset
+        - local: semantic_cache_chroma_vector_database
+          title: Implementing semantic cache to improve a RAG system.
+        - local: structured_generation
+          title: RAG with source highlighting using Structured generation
 
-- title: Computer Vision
-  sections:
-  - local: fine_tuning_vit_custom_dataset
-    title: Fine-tuning a Vision Transformer Model With a Custom Biomedical Dataset
+    - title: Computer Vision Recipes
+      isExpanded: false
+      sections:
+        - local: fine_tuning_vit_custom_dataset
+          title: Fine-tuning a Vision Transformer Model With a Custom Biomedical Dataset
 
-- title: Agents
-  sections:
-  - local: agents
-    title: Build an agent with tool-calling superpowers using Transformers Agents
+    - title: Diffusion Recipes
+      isExpanded: false
+      sections:
+        - local: stable_diffusion_interpolation
+          title: Stable Diffusion Interpolation
+
+    - title: Multimodal Recipes
+      isExpanded: false
+      sections:
+        - local: analyzing_art_with_hf_and_fiftyone
+          title: Analyzing Artistic Styles with Multimodal Embeddings
+        - local: faiss_with_hf_datasets_and_clip
+          title: Embedding multimodal data for similarity search
+
+    - title: Agents Recipes
+      isExpanded: false
+      sections:
+        - local: agents
+          title: Build an agent with tool-calling superpowers using Transformers Agents
 
 - title: Enterprise Hub Cookbook
+  isExpanded: False
```

Comment:
I think it'd be better to leave this expanded so users can quickly click on the "Overview" to see how this is different from the open-source cookbook.